src/integration-test/java/com/mongodb/jdbc/integration/MongoSQLIntegrationTest.java

```diff
@@ -55,4 +78,91 @@ public void loadTestConfigs() throws IOException {
         }
         return dynamicTests;
     }
+
+    @Test
+    public void testTypesOther() throws SQLException, IOException, IllegalAccessException {
```

Comment:
Now that we can load this type of data using a yaml file in the testdata directory, why do we have a manual test written out? Can't this test be written as a yaml test in the tests directory?

Reply:
I made the changes to process the document from the yaml file.  I was having trouble with getting the row to fit in document form.  I set the key as the index of the column and add it to the List<Object> to compare with the actual row.  That seems to work fine.